src/main/kotlin/com/terraformation/backend/search/field/AliasField.kt

```diff
@@ -24,7 +24,13 @@ private constructor(
      * The underlying field. Other than the field name, all the properties on the alias delegate to
      * this.
      */
-    val original: SearchField
+    val original: SearchField,
+
+    /**
+     * Whether this field can be exported to a CSV file. Only default (localized) fields may be
+     * requested for CSV export, since CSV files need to match what’s shown onscreen.
+     */
+    override val exportable: Boolean = true,
```

Comment:
Maybe there's a case here that I'm overlooking, but for alias fields, I think it might make sense to always delegate to the original field here (that is, get rid of this argument entirely and let the `by original` take care of passing the call through). My thinking is that exportability is really a property of the rendering format, which is always going to be determined by the underlying field.